wasmtime-interface-types/src/lib.rs

```diff
@@ -100,6 +100,20 @@ impl ModuleData {
         })
     }
 
+    /// Detects if WASI support is needed: returns module name that is requested.
+    pub fn find_wasi_module_name(&self) -> Option<String> {
+        self.inner.as_ref().and_then(|Inner { module }| {
+            module
+                .imports
+                .iter()
+                .find(|walrus::Import { module, .. }| match module.as_str() {
+                    "wasi" | "wasi_unstable" => true,
```

Comment:
WASI will have its first new snapshot soon, which will contain modules named "wasi_snapshot_*".

Reply:
Is wasi and wasi_unstable needed here after the snapshot? Is it intended for all modules to be wasi_unstable_<something>?

Reply:
Thanks. We probably will need to change logic of embedder to properly select right version of API implementation based on name. At this moment I choose return name, but later we can change `find_wasi_module_name` to return an enum.